### PR TITLE
quick sort on linked list

### LINKED ISSUE #875: quick sort on linked list in c++, python

### PR DESCRIPTION
**Pull request by raghav-dalmia**
Fixes #875 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- write code of quicksort on the linked list in c++ and python

#### Languages Used:
- c++
- python

#### Files Added:
- Linked_List_Quick_Sort.cpp
- Linked_List_Quick_Sort.py

Fixes #875 
@jainaman224 please review this PR.

### CODE REVIEW

File: Linked_List_Quick_Sort/Linked_List_Quick_Sort.cpp [hunk 1]
```diff
@@ -0,0 +1,144 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+class node
```
Comment (jainaman224) [inline Linked_List_Quick_Sort/Linked_List_Quick_Sort.cpp]: Class Name should be `Node`.

Comment (raghav-dalmia) [reply Linked_List_Quick_Sort/Linked_List_Quick_Sort.cpp]: @jainaman224 done

File: Linked_List_Quick_Sort/Linked_List_Quick_Sort.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# node class for singly linked list
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+
+# class for maintaining linked list
+class linkedList:
```
Comment (jainaman224) [inline Linked_List_Quick_Sort/Linked_List_Quick_Sort.py]: Class name should be `LinkedList`.

Comment (raghav-dalmia) [reply Linked_List_Quick_Sort/Linked_List_Quick_Sort.py]: done @jainaman224

**Comment by raghav-dalmia** [comment]
@jainaman224 please review this PR, if it requires any changes then let me know.

**Comment by raghav-dalmia** [comment]
pinging @jainaman224

**Comment by jainaman224** [comment]
Squash your commits and it is ready to be merged.

**Comment by raghav-dalmia** [comment]
@jainaman224 done, please merge this pr.

**Comment by raghav-dalmia** [comment]
> Squash your commits and it is ready to be merged.

@jainaman224 done

**Pull request merged**